README.md

```diff
@@ -1,2 +1,56 @@
-# cat-bnb
-Getting your cat the attention it deserves
+# Project Title
+
+## **Cat-BNB** 
+**Getting your cat the attention it deserves!**
+
+After reviewing the Swedish cyberspace, we discovered that there is no affordable solution for cat owners that want to leave their cats to a safe place, when going on vacation.
+
+During our studies in Craft Academy, we decided to make a small basic application which covers that need.
+
+Inspired by AirBNB, we set out on May 2019 to create something similar, where interested cat hosts can post their details, for cat owners to find them.
+
+This was a one week project (May 6th - 12th).
+
```

Comment:
i think we should add user stories here

Reply:
Yea, you are right. Forgot about them :P 
Added the section now. Will be adding them as soon as they are merged ;)

Reply:
Not sure about that. I would explain the main features and link to PT. There's a danger that this document becomes too big. It's your call though.

Reply:
That's a thought also. Thanks Thomas.